drivers/storage/portworx/component/disruption_budget.go

```diff
@@ -100,31 +100,41 @@ func (c *disruptionBudget) createPortworxPodDisruptionBudget(
 	cluster *corev1.StorageCluster,
 	ownerRef *metav1.OwnerReference,
 ) error {
-	var err error
-	c.sdkConn, err = pxutil.GetPortworxConn(c.sdkConn, c.k8sClient, cluster.Namespace)
+	userProvidedMinValue, err := pxutil.MinAvailableForStoragePDB(cluster)
 	if err != nil {
-		return err
+		logrus.Warnf("Invalid value for annotation %s: %v", pxutil.AnnotationStoragePodDisruptionBudget, err)
```

Comment:
Recommend also "nullifying" the `userProvidedMinValue`, since you use it 3 lines down -- i.e.

```go
	if err != nil {
		logrus.Warnf("Invalid value for annotation %s: %v", pxutil.AnnotationStoragePodDisruptionBudget, err)
		userProvidedMinValue = -1
	}
```